libs/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ include(HPX_Message)
 include(HPX_AddPseudoDependencies)
 include(HPX_AddPseudoTarget)
 
-set(HPX_LIBS
+set(HPX_CANDIDATE_LIBS
```

Comment:
Are we sure that `HPX_LIBS` is not used anywhere outside of HPX to refer to the list of modules (APEX)?